frontend/src/scenes/plugins/edit/PluginSource.tsx

```diff
@@ -12,40 +12,41 @@ import SCAFFOLD_errors from '!raw-loader!@posthog/plugin-scaffold/dist/errors.d.
 // @ts-ignore
 import SCAFFOLD_types from '!raw-loader!@posthog/plugin-scaffold/dist/types.d.ts'
 
-const defaultSource = `// Learn more about plugins at: https://posthog.com/docs/plugins/overview
-import { Plugin } from '@posthog/plugin-scaffold'
+const defaultSource = `
+// Learn more about plugins at: https://posthog.com/docs/plugins/build/overview
 
-type MyPluginType = Plugin<{
-  config: {
-    username: string
-  },
-  global: {},
-}>
+/* Runs on every event */
```

Comment:
This description could be a bit confusing, because we also have `onEvent` which is the one intended for running _on_ event. `processEvent` should only be used for actually processing/transforming the event in flight.